packages/voorbeeld-design-tokens/documentation/number-badge/utrecht.numberBadge.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { BadgeCounter } from '@utrecht/component-library-react/dist/css-module';
+
+const meta = {
+  id: 'utrecht-badge-counter-group',
+  title: 'Components/Number Badge/Utrecht',
```

Comment:
Kan je hier nog `Components/Number badge/Utrecht` van maken? Met kleine letter b in badge.